### ISSUE TITLE
Missing parameter 'shipmonkBaselinePerIdentifier.directory'.

### ISSUE THREAD

**Issue by ruudk** [shipmonk-rnd/phpstan-baseline-per-identifier#23]
I followed the readme (non legacy, using 2.1.0 and phpstan 2.0) and getting:

```
❯ composer run generate:baseline:phpstan
> phpstan --generate-baseline=src-dev/PHPStan/baselines/loader.php

In Helpers.php line 62:

  Missing parameter 'shipmonkBaselinePerIdentifier.directory'.

```

**Comment by janedbal** [comment]
What a stupid bug, thank you! Fixed in 2.1.1

**Comment by ruudk** [comment]
Thanks for the fast fix ⚡
